state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -626,14 +643,38 @@ pub mod pallet {
 		DepositBoosted {
 			deposit_address: TargetChainAccount<T, I>,
 			asset: TargetChainAsset<T, I>,
-			amount: TargetChainAmount<T, I>,
+			amounts: BTreeMap<BoostPoolTier, TargetChainAmount<T, I>>,
```

Comment:
Minor nit: I think we should consider changing this into a `Vec<(BoostPoolTier, TargetChainAmount)>`, it's a bit more json-friendly (although this is SCALE so maybe it doesn't matter). 

But also - doesn't the boost_id already uniquely identify the amounts?

Reply:
`BoostPoolTier` will be unique, that's why i explicitly used`BTreeMap`. I'm pretty sure SCALE will make it output the same as a vec. I prefer to leave it as is. Can change it later if we find its not suitable.

Reply:
Yeah that's fine. I was just imagining a future where we have more Tiers, or have more complex tiers (for example a tuple of `(Price, MinConfirmations)`). Just something to keep to in mind. WE can change it later if needed.